server/controllers/request.js

```
@@ -71,19 +77,24 @@ exports.updateRequest = asyncHandler(async (req, res, next) => {
   }
 });
 
-// @route PATCH /requests/:id/accept
+// @route PATCH /requests/accept/:id
 // @desc accept or decline request by sitter
 // @access Private
 exports.updateRequestAccepted = asyncHandler(async (req, res, next) => {
   const id = req.params.id;
   const { accepted, declined } = req.body;
   try {
-    const request = await Request.findOne({ _id: id });
-    if (request.sitter.toString() === req.user.id) {
+    const request = await Request.findById(id);
+    const user = await User.findById(req.user.id).populate('profile');
+    // only sitter can accept or decline
+    if (request.sitter.toString() === user.profile._id.toString()) {
```

if they are not the same, should we return an error here?